src/components/SignUp.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import mentors from "../images/mentor.jpg";
+import Login from "./Login";
+import SignUpForm from "./SignUpForm";
+export default class SignUp extends Component {
+  tabHandler = (evt, cityName) => {
+    var i, tabcontent, tablinks;
```

Comment:
can we use **let** instead of **var**?

Reply:
you should either use just es5 or just es6, try not to mix things up

you can use let

Reply:
yes. ok